src/main/java/dk/cngroup/trainings/spring/springassignment/controller/security/WebSecurityConfig.java

```diff
@@ -0,0 +1,42 @@
+package dk.cngroup.trainings.spring.springassignment.controller.security;
```

Comment:
the convention is not to have this in controller package (even though I kinda understand the reason, beacause you are handling the endpoints here). You are setting the usernames and passwords here, so it does not belong here.

Usually all Config files for Spring (where you define @Beans etc) go to config package on top level.

Reply:
I will move them.